Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -89,6 +89,8 @@ FullDiagnostics::ReadParameters ()
     raw_specified += pp_diag_name.query("plot_raw_fields_guards", m_plot_raw_fields_guards);
 
 #ifdef WARPX_DIM_RZ
+    if( m_format == "openpmd" )
+        m_dump_rz_modes = 1;
```

Comment:
I cannot quickly spot where we use that option yet @dpgrote :)

Ideally, I would like to disable re-construction of fields in that case.

Alternatively, can we rename reconstructed fields to read `<fieldname>theta=0` instead of `<fieldname>` to be more explicit? :)

Reply:
oh found it, nevermind. it's in `FullDiagnostics::AddRZModesToDiags`